nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/CpuBackend.java

```diff
@@ -18,6 +18,7 @@
 
 import org.nd4j.linalg.factory.Environment;
 import org.nd4j.linalg.factory.Nd4jBackend;
+import org.nd4j.nativeblas.Nd4jCpu;
```

Comment:
Let's try to avoid exposing this directly. You can use Nd4j.getNativeOpsHolder().getDeviceNativeOps().buildInfo();
instead here. Do this for both cpu and cuda. You can put this in the base class and maintain the abstraction.

Reply:
sure, I will move it to NativeOps as desired. 
I would like also to note that,imho, buildInfo fits here more. 
And as adding an extra import for Nd4jCpu is not good,  still, its sister side breaks it.  JCublasNd4jCuda is already [has import Nd4jCuda](https://github.com/KonduitAI/deeplearning4j/blob/3999b85514148ec63337c8c15cde17f6e4cede1a/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-cuda/src/main/java/org/nd4j/linalg/jcublas/JCublasBackend.java#L108)

Reply:
@saudet 
as I want to introduce buildInfo as non-static function I had to add `.put(new Info("build_info.h").objectify())`
but it strangly corrupts all statics. any thoughts?

Reply:
Not sure what you mean by that, but try to add it to this line instead:
https://github.com/KonduitAI/deeplearning4j/blob/master/nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/nativeblas/Nd4jCpuPresets.java#L164

Reply:
yes I'm adding there. 

> @saudet
> as I want to introduce buildInfo as non-static function I had to add .put(new Info("build_info.h").objectify())
> but it strangly corrupts all statics. any thoughts?